[Snackbar] Flaky color themer test

### Overview
Seen in kokoro bazel build (Xcode 9.0)

```
ERROR: /components/Snackbar/BUILD:91:1: Compiling Swift module unit_test_swift_sources (3 files) failed (Exit 1)
components/Snackbar/tests/unit/MDCSnackbarColorThemerTests.swift:19:8: error: no such module 'components_Snackbar_ColorThemer'
import components_Snackbar_ColorThemer
       ^
```

Seen in another kokoro bazel build (Xcode 9.1)

```
ERROR: /components/Snackbar/BUILD:91:1: Compiling Swift module unit_test_swift_sources (3 files) failed (Exit 1)
components/Snackbar/tests/unit/MDCSnackbarColorThemerTests.swift:19:8: error: no such module 'components_Snackbar_ColorThemer'
import components_Snackbar_ColorThemer
       ^
```

--- a/components/ActivityIndicator/BUILD
+++ b/components/ActivityIndicator/BUILD
@@ -58,7 +58,7 @@ mdc_objc_library(
     ],
     deps = [
         ":ActivityIndicator",
-        "//components/Schemes/Color",
+        "//components/schemes/Color",
     ],
     visibility = ["//visibility:public"],
 )

--- a/components/BottomNavigation/BUILD
+++ b/components/BottomNavigation/BUILD
@@ -58,7 +58,7 @@ mdc_objc_library(
     ],
     deps = [
         ":BottomNavigation",
-        "//components/Schemes/Color",
+        "//components/schemes/Color",
     ],
     visibility = ["//visibility:public"],
 )

